server/routes.go

```diff
@@ -586,3 +587,38 @@ func streamResponse(c *gin.Context, ch chan any) {
 		return true
 	})
 }
+
+func initServer() {
+	// calling these loads the runners into a temporary directory
+	llm.GGMLRunners()
+	llm.GGUFRunners()
+}
+
+func cleanupServer() {
+	if loaded.llm != nil {
+		loaded.llm.Close()
+	}
+	var tmpDirs []string
+	if len(llm.GGMLRunners()) > 0 {
```

Comment:
It might be easier to set up a top-level ollama working directory and store everything in there, e.g. `/tmp/ollamaXXXXXX/{ggml,gguf}/server`